central/processlisteningonport/datastore/datastore_impl.go

```diff
@@ -271,11 +272,11 @@ func (ds *datastoreImpl) GetProcessListeningOnPort(
 ) (
 	processesListeningOnPorts []*storage.ProcessListeningOnPort, err error,
 ) {
-	if ok, err := plopSAC.ReadAllowed(ctx); err != nil {
-		return nil, err
-	} else if !ok {
-		return nil, sac.ErrResourceAccessDenied
-	}
+	// if ok, err := plopSAC.ReadAllowed(ctx); err != nil {
```

Comment:
The check for access has been moved to `ds.storage.GetProcessListeningOnPort`. Perhaps there should be a check here as well.

Reply:
I will not have the check here.